crp_APL/plan_motion_planning/src/motionHandler.cpp

```diff
@@ -18,17 +14,38 @@ crp::apl::MotionHandler::MotionHandler() : Node("motion_handler")
 
     m_pub_trajectory_ = this->create_publisher<autoware_planning_msgs::msg::Trajectory>("plan/trajectory", 10);
 
+    m_timer_ = this->create_wall_timer(std::chrono::milliseconds(20), std::bind(&MotionHandler::run, this));
 }
 
 void crp::apl::MotionHandler::scenarioCallback(const tier4_planning_msgs::msg::Scenario::SharedPtr msg)

```

Comment:
This should be renamed to strategy callback. Scenario is the message type. About that wouldn't std_msgs/msg/String suffice for this message?
```suggestion
void crp::apl::MotionHandler::strategyCallback(const tier4_planning_msgs::msg::Scenario::SharedPtr msg)

```